aeneas/src/x86/X86CodeGen.v3

```diff
@@ -670,6 +794,21 @@ class X86CodeGen extends MachCodeGen {
 		m(asm, r(dest), r(disp).plus(X86Addrs.ABS_CONST));
 		recordPatch(pos, addr);
 	}
+	def asm_loadx_sse(m: (X86Assembler, SSEReg, SSERm) -> void, dest: int, base: int, scale: byte, disp: int, canTrap: bool, source: Source) {
+		var off = asm.codeOffset();
+		m(asm, s(dest), SSEAddr.new(null, r(base), scale, disp));
+		if (canTrap && rtsrc != null) rtsrc.recordSource(off, source);
```

Comment:
That reminds me...we should have stack trace tests for OOB/nullchecks involving floats and doubles. Can you drop a TODO somewhere (I guess here is fine) that we need to add that to the stacktrace tests? It's a bit tricky because floating point is still an optional feature and the stacktrace tests don't deal with that yet.